pom.xml

```diff
@@ -103,6 +103,16 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>com.cedarsoftware</groupId>
+                <artifactId>java-util</artifactId>
+                <version>1.8.0</version>
```

Comment:
Move the versions to parent pom.

Reply:
[Fixed commit](https://github.com/wso2-extensions/identity-fetch-remote/pull/12/commits/e33283f45ce3ac37a3c2ef622918b08c838285d4)